homeassistant/components/climaveneta_imxw/strings.json

```
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configure Climaveneta iMXW",
+        "description": "Insert the Home Assistant modbus hub and the address ID of the iMXW",
+        "data": {
+          "hub": "Modbus hub name",
+          "slave": "Address ID",
+          "name": "Name"
```

You can use references here 
```suggestion
          "name": "[%key:common::config_flow::data::name%]"
```